src/content/docs/en/reference/dev-overlay-plugin-reference.mdx

```diff
@@ -223,7 +223,7 @@ Shows a card with optionally an [`icon`](#icons). Optionally, if a `link` is pas
 The slot of the component will be used as the content of the card.
 
 ```html
-<astro-overlay-card icon="astro:logo" link="https://github.com/withastro/astro/issues/new/choose">Report an issue</astro-overlay-card>
+<astro-dev-overlay-card icon="astro:logo" link="https://github.com/withastro/astro/issues/new/choose">Report an issue</astro-dev-overlay-card>
```

Comment:
I have to comment to see if the bot notices this.  😆

Reply:
narrator: it did not
(it did)